pkg/sync/rpc/server.go

```diff
@@ -362,7 +369,8 @@ func (s *SyncAgentServer) launchReceiver(processName, toFileName string, ops spa
 		}()
 
 		logrus.Infof("Running ssync server for file %v at port %v", toFileName, port)
-		if err = sparserest.Server(strconv.Itoa(port), toFileName, ops); err != nil && err != http.ErrServerClosed {
+		directIO := true
+		if err = sparserest.Server(strconv.Itoa(port), toFileName, ops, directIO); err != nil && err != http.ErrServerClosed {
```

Comment:
Wait... Is it meaningful to add the `directIO` flag in the receiver side? Should the sender tell the receiver if it needs to use direct io to write the data into disk?

Reply:
yes, with mod, it no longer make sense to have it on the server side. I have removed it and wait for the integration result.